lxc/info.go

```diff
@@ -507,18 +507,30 @@ func (c *cmdInfo) instanceInfo(d lxd.InstanceServer, name string, showLog bool)
 		fmt.Printf("  "+i18n.G("Processes: %d")+"\n", inst.State.Processes)
 
 		// Disk usage
-		diskInfo := ""
+		diskUsage := ""
+		diskTotal := ""
 		if inst.State.Disk != nil {
 			for entry, disk := range inst.State.Disk {
-				if disk.Usage != 0 {
```

Comment:
interesting that we have this logic here, so this is an API break?

Reply:
And this commit specifically is now breaking this lxc client talking with older servers that return 0 for no disk usage.

Makes me thing we should be sticking with zero then.

Reply:
Because this is clearly not unintended behaviour, but was explicitly intended to be this way by the looks of it.

I think it would be good to introduce a constant in the api package that we can return and test for with a value of 0, to indicate unsupported disk usage.

Reply:
I think there is some confusion on what is going on here. Currently, when the driver does not support getting the usage, `inst.State.Disk` is an empty map. So this commit does not break the client from talking with older servers, I just tested this to be sure.

Reply:
That said, if you would still like to switch to using 0, I am fine with it.

Reply:
The only way to get 0 usage is if using a mock backend, and this was already the case when this check was added. I don't see how but could that be the reason for that check?

If not, maybe this was just hiding a value that was definetely wrong.

Reply:
@tomponline If this is fine, this PR should be ready to merge